src/pages/index.js

```diff
@@ -59,11 +59,11 @@ api
     console.error(err);
   });
 
-api
-  .getUserInfo()
-  .then((userData) => {
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([userData, initialCards]) => {
     userInfo.setUserInfo(userData.name, userData.about);
     userInfo.setAvatar(userData.avatar);
+    initialCards = userData;
```

Comment:
Needs correction: Please move the code from lines 46-56 for initializing the cards section to here, but don't forget to rename `res` to `initialCards`